pom.xml

```diff
@@ -189,6 +196,14 @@
             <version>4.4</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>23.0</version>
+        </dependency>
+        <!-- Thanks for using https://jar-download.com -->
```

Comment:
Nit: Not rerquired

Reply:
It required by Unit Test.

Reply:
I meant, L204, the comment. Apologies for not being clear.
 `<!-- Thanks for using https://jar-download.com -->`

Reply:
OK. I will remove it.